db/migrate/20200217061622_add_timestamp_to_news.rb

```diff
@@ -0,0 +1,9 @@
+class AddTimestampToNews < ActiveRecord::Migration[6.0]
+  def change
+    add_column :news, :updated_on, :datetime
```

Comment:
I now believe that it should be called updated_at instead of _on. We could then take the chance to also rename created_on to created_at.